main.go

```diff
@@ -1,18 +1,18 @@
 package main
 
 import (
-	gpc "./GroupOfCells"
+	"github.com/LukeJelly/Cell-Compete-Go/groupofcells"
```

Comment:
It's best to use fully-qualified packages so that things are easy to distribute and re-use in other projects. You'll see what I mean when I import a testing library in the next PR.